apps/yggdrasil_web/web/controllers/character_controller.ex

```diff
@@ -64,4 +86,8 @@ defmodule YggdrasilWeb.CharacterController do
         render conn, :errors, data: err_changeset
     end
   end
+
+  defp admin?(conn) do
+    EnsurePermission.all?(conn, character: [:admin])
```

Comment:
@copenhas I'm really ok with either set of permissions at this point, it should be very easy to change these as we need to. The plug doesn't make any assumptions about them so it's really just boils down to going back and changing them in them in the controllers.

For now I renamed :admin to :all, but only because it was better than :admin and I didn't want to change to much to there is some consensus on the set we want.